src/lib.rs

```diff
@@ -10,14 +10,14 @@ use kubewarden::{protocol_version_guest, request::ValidationRequest, validate_se
 mod settings;
 use crate::settings::Rule;
 
-const ALL_ARE_USED_ERROR_MSG: &str =
-    "Resource cannot have all the environment variables from the rule defined.";
-const NOT_ALL_ARE_USED_ERROR_MSG: &str =
-    "Resource should have all the environment variables from the rule defined. Invalid environment variables found:";
 const ANY_IN_ERROR_MSG: &str =
-    "Resource cannot define any of the environment variables from the rule. Invalid environment variables found:";
+    "Resource must have at least one of the required environment variables specified by the validation rule. None of the expected environment variables were found:";
 const ANY_NOT_IN_ERROR_MSG: &str =
-    "Resource should define at least one of the environment variables from the rule. Invalid environment variables found:";
+    "Resource must not have any of the environment variables specified in the validation rule. The following invalid environment variables were found:";
+const ALL_ARE_USED_ERROR_MSG: &str =
+    "Resource is missing required environment variables as specified in the validation rules. The following environment variables are missing:";
+const NOT_ALL_ARE_USED_ERROR_MSG: &str =
+    "Resource has conflicting environment variables set according to the validation rules. The following environment variables should not be set together:";
```

Comment:
Please, rename the constants to match the new keywords. Like `anyIn` -> `containsAnyOf`